.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: release
+on:
+  push:
+    tags:
+    - 'v*.*.*'
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@master
+    - name: Create Release
```

Comment:
I actually think you don’t need to create a release. Every tag is sort of implicitly is a release. And we don’t upload files etc. yet the download link would still work without a release.

What do you think? Would that work with the bot?

Reply:
the bot at this moment expect release to be there. It also expected assets, which I removed while working on PR for kubectx. 

I think we can relax not having release too. I will need to make changes to action/bot for that. and I will do that this weekend.

Reply:
I'm not really against creating a release, I guess it's ok to have this in this repo.

But I suspect many other bash-based repos will not bother creating releases. git tag && git push is good enough for many.